src/clj/clojurians_log/routes.clj

```diff
@@ -6,11 +6,13 @@
             [clojurians-log.views :as views]
             [clojurians-log.slack-messages :as slack-messages]
             [clojurians-log.time-util :as time-util]
+            [clojurians-log.routes-def :refer [routes]]
             [java-time :as jt]
-            [compojure.core :refer [GET routes]]
             [compojure.route :refer [resources]]
             [datomic.api :as d]
-            [ring.util.response :refer [response]]))
+            [ring.util.response :refer [response]]
+            [bidi.ring :as bidi.r :refer (make-handler)]
```

Comment:
as per the style guide, use a vector for the `:refer []`

Reply:
or use `bidi.ring/make-handler`, since this is the main place where we hook up bidi it'd be good to make it clear that this is a bidi function.